pkg/synthetictests/duplicated_events_test.go

```diff
@@ -475,24 +467,21 @@ func TestMakeProbeTestEventsGroup(t *testing.T) {
 				events = append(events,
 					monitorapi.Interval{
 						Condition: monitorapi.Condition{Message: message},
-						From:      time.Unix(1, 0),
-						To:        time.Unix(1, 0)},
+						From:      time.Unix(1, 0).In(time.UTC),
+						To:        time.Unix(1, 0).In(time.UTC)},
 				)
 			}
 
 			junits := makeProbeTest("test test", events, test.operator, test.regEx, duplicateevents.DuplicateEventThreshold)
 
-			if len(junits) < 1 {
-				t.Fatal("didn't get junit for duplicated event")
-			}
+			assert.GreaterOrEqual(t, len(junits), 1, "Didn't get junit for duplicated event")
 
-			if test.match && strings.Compare(junits[0].FailureOutput.Output, test.expectedMessage) != 0 {
-				t.Fatalf("expected case to match, but it didn't: %s.  Expected:\n%s\nReceived:\n%s\n", test.name, test.expectedMessage, junits[0].FailureOutput.Output)
+			if test.match {
+				assert.Equal(t, test.expectedMessage, junits[0].FailureOutput.Output, "Expected case to match, but it didn't: %s. Expected:\n%s\nReceived:\n%s\n", test.name, test.expectedMessage, junits[0].FailureOutput.Output)
```

Comment:
Don't need to highlight the expected vs actual in your message, that is one of the main benefits of assert library, it gives consistent comparison of what differed for lots of objects.